frontend/src/utils/gameDataUtils.js

```diff
@@ -0,0 +1,18 @@
+export const getPlayerData = (game, playerId) =>
+  game.betSession.players.find((player) => player.id === playerId);
+
+export const getMinBet = (game, playerId) => {
+  const playerBet = getPlayerData(game, playerId).currentBet;
+  return (
+    Math.max(...game.betSession.players.map((player) => player.currentBet)) -
+    playerBet
+  );
+};
+
+export const isActive = (game, playerId) =>
+  playerId === game.betSession.players[game.betSession.activePlayerIndex].id;
+
+export const getPot = (game) =>
+  game.betSession.players
+    .map((player) => player.currentBet)
+    .reduce((sum, currentBet) => sum + currentBet);
```

Comment:
```suggestion
export const getPot = (game) =>
  game.betSession.players
    .reduce((sum, player) => sum + player.currentBet, 0);
```
🤣 I just love the reduce method

Reply:
Unfortunatley you suggestion does not work here, because js tries to add an int to an object type...